### DISCUSSION TITLE
Upgrade from 0.9.9 to the last version 0.13.0 ?

### DISCUSSION THREAD

**Discussion by inetactief** [AzuraCast/AzuraCast#4429] [General]
Hi, 

We running for a long time on 0.9.9 and we want to upgrade to the last version. 
Is this a problem or should I do this in stages?

Best regards,
Michel

v0.9.9, #5903387 (2020-04-07 4:54) • Docker • PHP 7.4  (Running on Centos 7)

**Comment by BusterNeece** [comment]
@inetactief Definitely take a backup of your install before updating, but it can and should be done all at once from that version to the current one. It may take a bit to get through some of the migrations but it should do it no problem.

**Comment by inetactief** [comment]
Hi,

Thank you the upgrade works. Not direct, the ssl cert. do not work the first time, but after restarting and update again it works.

Nice work, the new version! 😊

Best regards,
Michel



 

Van: SlvrEagle23 ***@***.***> 
Verzonden: maandag 26 juli 2021 20:10
Aan: AzuraCast/AzuraCast ***@***.***>
CC: Michel ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [AzuraCast/AzuraCast] Upgrade from 0.9.9 to the last version 0.13.0 ? (#4429)

 

@inetactief <https://github.com/inetactief>  Definitely take a backup of your install before updating, but it can and should be done all at once from that version to the current one. It may take a bit to get through some of the migrations but it should do it no problem.
